packages/schematics/angular/utility/find-module_spec.ts

```diff
@@ -74,6 +74,14 @@ describe('find-module', () => {
         expect(err.message).toMatch(/Could not find an NgModule/);
       }
     });
+
+    it('should accept custom ext for module', () => {
```

Comment:
Could you add a falsifying test that checks that it doesn't match previous files when using the new arguments?